source/options.css

```diff
@@ -16,15 +16,29 @@ ul {
 	padding-left: 1.5em;
 }
 
+[data-scope] {
+	float: right;
+	margin-right: 50px;
+}
+
+.invalid [data-scope] {
+	display: none;
+}
+
 :root [name='customCSS'],
 :root [name='personalToken'] {
 	font-family: SFMono-Regular, Consolas, 'Liberation Mono', Menlo, Courier, monospace;
 	font-size: 11px;
 	line-height: 1.5;
 }
 
+#validation {
+	display: inline-block;
+}
+
 [name='personalToken'] {
 	width: 20em !important; /* https://github.com/sindresorhus/refined-github/issues/1374#issuecomment-397906701 */
+	display: inline-block !important;
```

Comment:
Why is this important?

Reply:
Needed so that the span displays next to it